01-Login/README.md

```diff
@@ -68,4 +75,1059 @@ If you have found a bug or if you have a feature request, please report them at
 
 ## License
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE.txt) file for more info.
+This project is licensed under the MIT license. See the [LICENSE](../LICENSE) file for more info.
+
+# Tutorial
+This tutorial will help you get up and running the React app using the new SDK.
+
+## Setting Up the Application
+
+### Installing initial dependencies
+After create a new React app using `create-react-app` we still need to install some dependencies that doesn't come with the boilerplate. These dependencies will help us with things like routing, styling and font icons. But, before that we need to move the current `dependencies` to `devDependencies` this is because we will be using a server to run the built app.
+
+```json
+- "dependencies": {
++ "devDependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "2.1.8"
+  }
+```
+
+After that we can install the new dev dependencies.
+
+```bash
+npm i -D react-router-dom highlight.js reactstrap samples-bootstrap-theme \
```

Comment:
For the tutorial, the visual aspects of the application can be dropped as it's a detail that's specific to the sample application.

Here, `samples-bootstrap-theme`, `highlight.js`, `reactstrap` and the FontAwesome dependencies can be dropped.